src/Console/Command/FixCommand.php

```diff
@@ -127,6 +128,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $passedConfig = $input->getOption('config');
         $passedRules = $input->getOption('rules');
 
+        $indent = $input->getOption('default-indentation');
+        Config::setDefaultIndent($indent);
```

Comment:
as you see with other options, options are passed to `ConfigurationResolver` to resolve their final value, after optional option and config consideration.

using this static hook is no go.

-----------------------

For now, we were accepting any `ConfigInterface` instance for this command.
Now, you bounded new input option with concrete implementation - `Config`.
**This new option won't work at all for all ppl using their custom config class.**

Reply:
> the whole tool is about standardization and unification, option to bypass it and have different rules for different files doesn't make me happy, personally

I understand your care about ideal standardization.  However, not all cases and user preferences perfectly match the strict model.  That's why it allow set custom rules to disable some PSR2 standardization, and have Config::setIndent().  Having necessary customization is good, so that the tool can be more useful for different people, and more widely used as a good code formatter.